vulnerabilities/data_dump.py

```diff
@@ -127,7 +127,17 @@ def archlinux_dump(extract_data):
             VulnerabilityReference.objects.create(
                 vulnerability=vulnerability,
                 reference_id=vulnerability_id,
+<<<<<<< HEAD
```

Comment:
You have an unresolved merge conflict here... but why would your npm code be related to archlinux code?

Reply:
This is because we both made some trivial changes to the same lines of the archlinux code. That's what I was referring to at the end of my earlier comment (https://github.com/nexB/vulnerablecode/pull/121#issuecomment-544880918):

> The conflicting code on both branches does the same thing, just with different syntax (string formatting with '{}'.format(...) vs f'{...}'). I prefer the f'{...}' syntax, but pick whichever one you want.